<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="mainStack">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="setup">
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>50</y>
        <width>791</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>48</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Steam Count</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="apiKeyInput">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
    background-color: #f2f2f2;
}

QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    padding: 5px;
    border-radius: 3px;
}

</string>
      </property>
      <property name="placeholderText">
       <string>Input API_KEY</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="steamIDInput">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>290</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
    background-color: #f2f2f2;
}

QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    padding: 5px;
    border-radius: 3px;
}

</string>
      </property>
      <property name="placeholderText">
       <string>Input STEAM_ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>390</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #66c0f4;
    color: #ffffff;
    border: none;
    padding: 8px;
    border-radius: 3px;
}

QPushButton:hover {
    background-color: #005fa3;
	transform: scale(0.8);
}</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="background">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>-5</y>
        <width>811</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url('./bg1.png');
background-position: center;
background-origin: content;
background-clip: border;
background-size: 100% 100%;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>background</zorder>
     <zorder>title</zorder>
     <zorder>apiKeyInput</zorder>
     <zorder>steamIDInput</zorder>
     <zorder>submitButton</zorder>
    </widget>
    <widget class="QWidget" name="main">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(27, 40, 56);</string>
     </property>
     <widget class="QLabel" name="currentGameText">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>801</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Currently Playing: NONE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="gameImage">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>461</width>
        <height>215</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="currentPlayerNumber">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>420</y>
        <width>801</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Current Players: NONE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="pfpImage">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>20</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="playerName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>671</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
